ddl/multi_schema_change_test.go

```diff
@@ -505,6 +505,127 @@ func TestMultiSchemaChangeAddDropIndexes(t *testing.T) {
 	tk.MustExec("admin check table t;")
 }
 
+func TestMultiSchemaChangeModifyColumns(t *testing.T) {
+	store, clean := testkit.CreateMockStore(t)
+	defer clean()
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test;")
+
```

Comment:
Could we add a test for modifying the same column, like `modify column a ..., modify column a ...`

Reply:
Sure.